astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2143,6 +2154,7 @@ def __init__(
         col_offset=None,
         parent=None,
         *,
+        doc_node: Optional[Const] = None,
```

Comment:
Same here, I think it would be good to move it after `end_col_offset`. Move the docstring too.